pkgs/top-level/all-packages.nix

```diff
@@ -29029,7 +29029,11 @@ with pkgs;
     withNetworkd = true;
     withLibidn2 = true;
   };
-
+  # remove when util-linux 2.39.3 comes out
+  systemdBcachefs = systemd.override {
```

Comment:
Shouldn't we just pull the util-linux patch? Not so sure about a new systemd…

Reply:
@RaitoBezarius I tried applying the patches but it wouldn't compile, I opened a bug report to get a stable release with patches applied, there's about 5 or so patches in master.

systemd hardcodes the mount paths to `util-linuxMinimal` it took me ages to figure out why `systemd-remount-fs` service would fail but everything else wouldn't.  this is the solution to that.

I was trying to avoid using util-linux from master on non-bcachefs nixpkgs, if you can figure out how to get util-linux to compile with just the minimal set of patches we can go with that.

Reply:
IMHO, this is a blocker because I already have patches on the top of systemd and this will make things much more complicated for developers like me who works on systemd.

We should find a way to put those patches inside the default util-linuxMinimal.